### ISSUE TITLE
Login won't stay persistent after signing in.

### ISSUE THREAD

**Issue by 1darklyte** [bokub/home-assistant-extension#6]
Using FF 88.0.1

After signing into HA using the add on it will not stay logged in. After a period of time (5+ min) it logs me out and asks me to log in again.

**Comment by bokub** [comment]
After you're logged out of, what happens if you visit Home Assistant directly? (not through the extension)

**Comment by 1darklyte** [comment]
My HA session accessed at the same URL is always connected via a browser window and stays logged in even when the extension logs me out. 

Also not sure if it helps but  the extension has me go through 2FA and I successfully complete that every time I log into the extension.

**Comment by johnjoemorgan** [comment]
Experiencing the same.  Not uses 2FA.  I have made sure all Firefox and other privacy settings are off for the HAS IP address.  I call the extension, login and use the HAS page.  I can go back a few moments or minutes later and I'm asked to log back in again.

**Comment by bokub** [comment]
I'm sorry but I cannot reproduce this on my side, so I am completely enable to investigate and fix it :disappointed: 

> 2. When the HEADER is made visible the bug doesn't seem to appear.

I **highly** doubt it, because I'm just cropping the window with CSS ! I don't see how it can influence anything related to authentication...

**Comment by johnjoemorgan** [comment]
Yep Agreed - seems that that was a false positive.  It's back to asking for the login.  Let me try again tomorrow on a different PC and see if I can get any clues in console for you.

**Comment by johnjoemorgan** [comment]
BTW - thanks for this.  This linked with the Spotify/Spotcast custom component and the Mini Media Player card is a fantastic combo.

**Comment by 1darklyte** [comment]
sry I guess I didn't get an email for this chain. Testing now with the header enabled to see if that changes anything.

**Comment by 1darklyte** [comment]
Well that took about 2 min for it to appear again haha. 

https://prnt.sc/134kfmf

I was actually in the middle of checking another bug I found which when it happened again.

2nd bug is (and im not sure if this is by design or a limitation) but when I attempt to use the header (now that I have exposed it), it wont change the page but it does move a long the top navigation.  https://prnt.sc/134khwv vs https://prnt.sc/134klu0

Also its now asked me to login 4 times in 6 minutes since I started testing again

**Comment by bokub** [comment]
Thanks for the screenshot, but I still don't know how to reproduce it on my side, so I cannot do anything :man_shrugging:

On my computer, I stay logged in 100% of the time, as it should be

**Comment by johnjoemorgan** [comment]
OK I haven't used Chrome since yesterday and opening it now the extension
works perfectly and I'm logged in.  Firefox 80.0.1 won't retain the login
for the extension.  I have all FF and other adblocks/privacy switched off

**Comment by johnjoemorgan** [comment]
I think I have the cause.  In Firefox about:preferences#privacy I had "Enhanced Tracking Protection" to STRICT.  Changing this to STANDARD allows the HAS login to persist.  Perhaps we can use the CUSTOM setting to avoid this. 

Anyhows, hope this helps .

**Comment by bokub** [comment]
That's it !

I tried to set the protection to "strict" and I the extension asked me to log in

Switched it again to "standard", and the extension was working again

Well done :+1:

**Comment by 1darklyte** [comment]
I wonder if there is a way to do this without lowering security settings for the whole browser.
